widget/button.go

```diff
@@ -578,6 +583,10 @@ func (b *Button) draw(screen *ebiten.Image) {
 	}
 
 	if i != nil {
+		if b.Image.Mask != nil && (b.MaskImage == nil || (b.MaskImage.Bounds().Dx() != b.widget.Rect.Dx() || b.MaskImage.Bounds().Dy() != b.widget.Rect.Dy())) {
```

Comment:
We could probably move this out of the draw loop and into the SetLocation method (which sets size and location)

Reply:
done